src/pages/Volunteering/Volunteer.jsx

```diff
@@ -116,7 +117,7 @@ function Volunteer() {
         <motion.div
           initial={{ opacity: 0 }}
           whileInView={{ opacity: 1 }}
-          transition={{ ease: "easeOut", duration: 4 }}
+          transition={{ ease: "easeOut", duration: TWO_SECONDS }}
```

Comment:
two seconds or four seconds?

there are many cases where this happens

please confirm if if all `4` should change to `TWO_SECONDS`